conf/common/socket.conf

```diff
@@ -39,7 +39,7 @@ socket_configuration: {
 	//	(Maximum Supported Connections)/2
 	// NOTE: this controls the maximum collected socket-events per-cycle (call to epoll_wait())
 	//       for example settings this to 32 will allow up to 32 events (incoming data/new connections
-	//       per server-cycle.
+	//       per server-cycle).
```

Comment:
this bracket should be moved to the line above so it's correct in both
`32 events (incoming data/new connections) per server-cycle.` and `32 events per server-cycle.`